microsoft/testsuites/dpdk/dpdksuite.py

```diff
@@ -89,6 +90,23 @@ def verify_dpdk_build_failsafe(
     ) -> None:
         self._verify_dpdk_build(node, log, variables, "failsafe")
 
+    @TestCaseMetadata(
+        description="""
+           Install and run ci test for NFF-Go on ubuntu
+        """,
+        priority=2,
+        requirement=simple_requirement(
+            min_nic_count=2,
+            network_interface=Sriov(),
+        ),
+    )
+    def verify_dpdk_nff_go(
+        self, node: Node, log: Logger, variables: Dict[str, Any]
+    ) -> None:
+        nff_go = node.tools[DpdkNffGo]
+        nff_go.run_test()
+        pass
```

Comment:
The `pass` doesn't need, if there is content in a method.

Reply:
oop u right, will fix